rediscluster/client.py

```diff
@@ -736,6 +745,33 @@ def renamenx(self, src, dst):
 
         return False
 
+
+    def pubsub_channels(self, pattern='*', aggregate=True):
```

Comment:
I went with the named arguments where possible, which allows tools like iPython to 'know' what the arguments are.

For `pubsub_channels`, `def pubsub_channels(self, *args, aggregate=True)` would work in Python3 and not Python2. I could use `*args, **kwargs` for all three but that looses the self-documenting of named keyword arguments.